deployments/deploy-ssh-taas.sh

```diff
@@ -489,7 +489,8 @@ spec:
     imagePullPolicy: Always
     securityContext:
       privileged: ${SECURITY_CONTEXT}
-    command: ["sh", "-c", "cd /bin && ./taas"]
+    command: ["sh", "-c"]
+    args: ["cd /bin && ./taas -spec-dir=$SPEC_DIR"]
```

Comment:
--spec-dir (double-dash)?

Reply:
It wont work here, for some reason if the command was ginkgo it would take the flag with `--`, but otherwise it understands only `-`

Reply:
Oh wow. Thank you.